### ISSUE TITLE
edger8r fails when compiling by opam 4.06

### ISSUE THREAD

**Issue by ZahraTarkhani** [intel/linux-sgx#204]
compiling 'Util.ml' with opam 4.06 fails! (opam 4.02 and 4.04 works fine)

**Comment by llly** [comment]
Yes. ocaml 4.06 is not supported. We will update edger8r to support ocaml from 4.01 to 4.6 in next major release. You can try #202 as a workaround.
